dwave/cloud/utils.py

```diff
@@ -801,3 +802,197 @@ def bqm_to_dqm(bqm):
         dqm.set_quadratic(u, v, biases)
 
     return dqm
+
+def estimate_qpu_access_time(solver,
```

Comment:
Let's make this a method on [`StructuredSolver`](https://github.com/dwavesystems/dwave-cloud-client/blob/442211d5279e9e6a96dc42175549caef66844970/dwave/cloud/solver.py#L732). It'll be in a good company there with similar methods like: `max_num_reads()`, `reformat_parameters()`, `check_problem()`.

`solver` becomes `self`.